### DISCUSSION TITLE
Is there a specific reason why installations have to go into "C:\local\" or can it be anywhere else?

### DISCUSSION THREAD

**Discussion by karl-police** [Chatterino/chatterino2#3491] [Q&A] [answered]
For theh build documentation https://github.com/Chatterino/chatterino2/blob/master/BUILDING_ON_WINDOWS.md

Is there a specific reason why installations have to go into "C:\local\" or can it be anywhere else?

**Comment by pajlada** [comment, marked as answer]
This is no longer relevant as we recommend the use of conan or vcpkg.
